src/ChangedEntity.php

```diff
@@ -17,6 +17,8 @@ class ChangedEntity
 {
     /**
      * @var string
+     *
+     * @phpstan-var class-string<T> $className
```

Comment:
Tu use T, u have to defining the template on the class, and not only the construct i think